Create New page

chrisdedman opened the pull request:
Added a button to create a new page on the sidebar.

PenguinBoi12 commented on src/app/(protected)/notebooks/[notebook_id]/[page_id]/page.tsx:
```diff
@@ -1,17 +1,21 @@
     notebook(id: $id) {
       id
       title
+      ownerId
```
We don't need to query `ownerId` and `ownerType`


Part of the beauty of using GraphQL. We can query only the data we need.

PenguinBoi12 commented on src/app/(protected)/notebooks/[notebook_id]/page.tsx:
```diff
@@ -1,17 +1,21 @@
     notebook(id: $id) {
       id
       title
+      ownerId
```
We don't need to query `ownerId` and `ownerType`

PenguinBoi12 reviewed (changes requested):
![image](https://github.com/Code-Society-Lab/cursif-web/assets/49426163/50458800-2580-4256-99b2-fe17eabddc36)
This should show the "pointer" cursor. 

In general it seem good but there's still weird behaviour about the page list.
![pages](https://github.com/Code-Society-Lab/cursif-web/assets/49426163/6e96ca72-76f4-4ada-b158-233c0587a2f1)

chrisdedman commented:
I fixed the issues based on your comments. However, I still had some trouble with the rendering of the array after clicking on the + button to create a new page. So, I used the ``location.reload();`` for now. I'm not sure why my array isn't rendered at the same time as I created the new page.

chrisdedman commented:
> I fixed the issues based on your comments. However, I still had some trouble with the rendering of the array after clicking on the + button to create a new page. So, I used the `location.reload();` for now. I'm not sure why my array isn't rendered at the same time as I created the new page.

Fixed

PenguinBoi12 reviewed (approved):
This looks good! I think it's ready to be merged.
